cpp/src/hash/concurrent_unordered_multimap.cuh

```diff
@@ -379,6 +379,9 @@ public:
      * @Synopsis  Inserts a (key, value) pair into the hash map
      * 
      * @Param[in] x The (key, value) pair to insert
+     * @Param[in] pass The pass (partition) number for the partitioned hash table
```

Comment:
I think above this in the general documentation area you need to explain what a partitioned hash table is and how it works.

Reply:
I added some description in 1291316, but the general algorithm description is beyond this PR, and will be included with the actual implementation of the partitioned hash table build. Currently, the changes are only touching the hash map class.

Reply:
Where is the hash table if not in this class?